### PR TITLE
Upgrading to rails 5.0.0

### PR DESCRIPTION
**Pull request by alexey-klimuk**

### CODE REVIEW

**Comment by alexey-klimuk** [comment]
[[TARIFF17] Ruby on Rails 5.1 upgrade (and related gems) / Ruby 2.4.1](https://trello.com/c/IbTSMYto/1864-tariff17-ruby-on-rails-51-upgrade-and-related-gems-ruby-241)

File: Gemfile [hunk 1]
```diff
@@ -1,35 +1,54 @@
...
 
 gem "aws-sdk", "~> 2"
 gem "aws-sdk-rails", ">= 1.0.1"
+
+
 gem "hashie", "~> 3.4"
 gem "multi_json", "~> 1.11"
-gem "pg"
-gem "sequel-rails", "~> 0.9", ">= 0.9.12"
+gem "yajl-ruby", "~> 1.2", require: "yajl"
```
Comment (matthewford) [inline Gemfile]: I've noticed there is an upgraded version of this gem:

https://github.com/chef/ffi-yajl

File: Gemfile [hunk 2]
```diff
@@ -1,35 +1,54 @@
 source "https://rubygems.org"
 ruby "2.3.3"
 
-gem "rails", "4.2.8"
+gem "rails", "5.0.0"
+
+
+gem "pg"
```
Comment (matthewford) [inline Gemfile]: we should peg this to a version to avoid an unintended upgrade

File: config/cable.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+development:
+  adapter: async
+
+test:
+  adapter: async
+
+production:
+  adapter: redis
+  url: redis://localhost:6379/1
```
Comment (matthewford) [inline config/cable.yml]: this needed?

File: Gemfile (hunk 1 above)
Comment (matthewford) [reply Gemfile]: Well it uses a different system library - so we should stick with the current gem, while we check if that is available on the CF PaaS

**Comment by alexey-klimuk** [comment]
[[TARIFF17] Ruby on Rails 5.1 upgrade (and related gems) / Ruby 2.4.1](https://trello.com/c/IbTSMYto/1864-tariff17-ruby-on-rails-51-upgrade-and-related-gems-ruby-241)

**Pull request closed without merging**